docs/src/lang/tuples.md

```diff
@@ -158,21 +158,23 @@ is picked with `\E t \in S_1 \X S_2`.
 **Example in TLA+:**
 
 ```tla
-  { "A", "B", "C" } \X (1..65535)
+  { "A", "B", "C" } \X (65533..65535)
     \* A set of tuples. Each tuple has two fields:
     \*   - field 1 has the value from the set { "A", "B", "C" }, and
-    \*   - field 2 has the value from the set 1..65535.
+    \*   - field 2 has the value from the set 65533..65535.
 ```
 
-**Example in Python:** TLA+ functions are immutable, so we are using [frozendict]:
+**Example in Python:**
 
 ```python
-  # the pythonic way: a set of python tuples (indexed with 0, 1, ...)
-  frozenset({ (n, p)
-                for n in { "A", "B", "C" } for p in range(1, 65535 + 1) })
-  # the TLA+ way
-  frozenset({ frozendict({ 1: n, 2: p })
-                for n in { "A", "B", "C" } for p in range(1, 65535 + 1) })
+>>> # the pythonic way: a set of Python tuples (indexed with 0, 1, ...)
+>>> tuples = { (n, p) for n in { "A", "B", "C" } for p in range(65533, 65535 + 1) }
+>>> tuples == {('C', 65533), ('B', 65535), ('A', 65533), ('A', 65534), ('B', 65534), ('C', 65535), ('A', 65535), ('C', 65534), ('B', 65533)}
+True
```

Comment:
Why switch to assigning the value to `tuples` here and testing for equality, instead of just using the output as in all other cases?

Reply:
Because the order is not defined in sets and the doctest will randomly fail since the output is not fixed.

Reply:
Ah yes! This is a good solution. Thanks :)